laurent22/joplin
# Chore: Fixes #9449: Mobile: Fix "useNativeDriver" warning

# Summary

`useNativeDriver` needs to be specified when creating an animation. See [the documentation](https://reactnative.dev/docs/next/animations#using-the-native-driver).

As [`useNativeDriver`](https://reactnative.dev/docs/next/animations#using-the-native-driver) defaults to `false` (with a warning), this pull request sets `useNativeDriver` to `false`.

# Testing

1. Click "sync"
2. Check the logs for a `useNativeDriver`-related warning

This has been tested on an Android 12 emulator.
